cmd/conf/chain.go

```diff
@@ -52,27 +56,43 @@ func (c *ParentChainConfig) Validate() error {
 	return c.Connection.Validate()
 }
 
+// lint:require-exhaustive-initialization
 type L2Config struct {
-	ID        uint64                   `koanf:"id"`
-	Name      string                   `koanf:"name"`
-	InfoFiles []string                 `koanf:"info-files"`
-	InfoJson  string                   `koanf:"info-json"`
-	DevWallet genericconf.WalletConfig `koanf:"dev-wallet"`
+	ID               uint64                   `koanf:"id"`
+	Name             string                   `koanf:"name"`
+	InfoFiles        []string                 `koanf:"info-files"`
+	InfoJson         string                   `koanf:"info-json"`
+	InitialL1BaseFee string                   `koanf:"initial-l1base-fee"`
```

Comment:
Something that I didn't notice in previous reviews.
This flag will be better placed at the InitConfig instead of L2Config.
WDYT?